superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -15,15 +20,37 @@
 
 class BertBenchmarkModel(torch.nn.Module):
     """The BERT model for benchmarking."""
-    def __init__(self, config, num_classes):
+    def __init__(self, config, num_classes, enable_fp8=False):
```

Comment:
Do we need this? For example, in future, when we add FP4, it will change the interface again. Therefore, shall we try FP8 directly?

Reply:
Can we get the information from configuration.

Reply:
updated